packages/web-react/src/components/FileUploader/README.md

```diff
@@ -381,7 +381,7 @@ and [escape hatches][readme-escape-hatches].
 | `label`              | `string`                             | —        | ✕        | Field label                                                                                                                                                     |
 | `labelText`          | `string`                             | —        | ✕        | Label for input in Drop zone                                                                                                                                    |
 | `linkText`           | `string`                             | —        | ✕        | Link text in input in Drop zone                                                                                                                                 |
-| `maxFileSize`        | `number`                             | 1000000  | ✕        | The maximum size of the uploaded file in bytes                                                                                                                  |
+| `maxFileSize`        | `number`                             | 1000000  | ✕        | The maximum size of the uploaded file in bytes. [Learn how file sizes are calculated](#understanding-file-size-in-bytes).                                       |
```

Comment:
**suggestion:** Maybe, it would be better to display this in **bold**.
```suggestion
| `maxFileSize`        | `number`                             | 1000000  | ✕        | The maximum size of the uploaded file in **bytes**. [Learn how file sizes are calculated](#understanding-file-size-in-bytes).                                       |
```